object_manager_test.go

```diff
@@ -61,6 +61,18 @@ func (c *fakeConnector) GetObject(obj IBObject, ref string, res interface{}) (er
 			*res.(*[]HostRecord) = c.resultObject.([]HostRecord)
 		case *ZoneDelegated:
 			*res.(*[]ZoneDelegated) = c.resultObject.([]ZoneDelegated)
+		case *RecordAAAA:
+			*res.(*[]RecordAAAA) = c.resultObject.([]RecordAAAA)
+		case *RecordPTR:
+			*res.(*[]RecordPTR) = c.resultObject.([]RecordPTR)
+		case *RecordTXT:
+			*res.(*[]RecordTXT) = c.resultObject.([]RecordTXT)
+		case *RecordMX:
+			*res.(*[]RecordMX) = c.resultObject.([]RecordMX)
+		case *RecordSRV:
+			*res.(*[]RecordSRV) = c.resultObject.([]RecordSRV)
+		case *RecordAlias:
+			*res.(*[]RecordAlias) = c.resultObject.([]RecordAlias)
 		}
```

Comment:
What are the use of above cases? all of these are being moved to a different file anyway, may have to remove Record A from the switch case here as well
i dont see any of the cases being used in this file. Correct me if i am wrong

Reply:
There a few test cases which illustrate GET operation on records using Name or any other search fields instead of passing Reference ID. These cases are used when Reference ID is not passed.